src/chainstate/stacks/index/node.rs

```diff
@@ -88,6 +87,7 @@ impl error::Error for CursorError {
 /// All numeric values of a Trie node when encoded.
 /// They are all 7-bit numbers -- the 8th bit is used to indicate whether or not the value
 /// identifies a back-pointer to be followed.
+// TODO: this should be a define_u8_enum.
```

Comment:
Wanna just knock this one out?  Or does it induce a lot of changes?

Reply:
Sure, it should be pretty simple.

Reply:
Updated in c0fb0418d10800e764f88694b24e12ef917cdf39